app/stacks/uk-south/applications-service/storage-account.tf

```diff
@@ -0,0 +1,22 @@
+resource "azurerm_storage_account" "function_storage" {
```

Comment:
I think we could create a single storage account in uk-west and make it geo-redundant ("GRS" replication type). What do you think about this? It's how we're configuring function storage on the back office, we pass the uk-west storage account as a dependency to our uk-south stack.

Reply:
I was in two minds about setting up this separate one so shared storage sounds good to me.